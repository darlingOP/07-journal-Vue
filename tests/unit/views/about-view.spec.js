import { shallowMount } from "@vue/test-utils";
import AboutView from '@/views/AboutView'

describe('pruebas en el About View', () =>{
  test('debe concordar con el snapshot', () => {
    
    const wrapper = shallowMount(AboutView)

    expect(wrapper.html()).toMatchSnapshot()
  });  
})