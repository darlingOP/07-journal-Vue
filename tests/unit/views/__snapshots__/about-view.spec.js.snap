// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en el About View debe concordar con el snapshot 1`] = `
<div>
  <h1>This is an about page</h1>
</div>
`;
